ircbot/plugins/transfer.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+from ircbot import bot
+from redis import StrictRedis
+
+
+@bot.command('transfer')
+def transfer(bot, channel, sender, args):
+    """"command to transfer karma & butts from one term to another"""
+
+    if len(args) == 2:
+            first_name = args[0].lower()
+            second_name = args[1].lower()
+    else:
+        string = args.join(" ")
+        strings = []
+        for term in re.findall("(?![\'\"])[a-z,\s]+(?=[\"\'])", string):
+            strings.append(term.strip())
+        try:
+            first_name = strings[0].lower()
+            second_name = strings[1].lower()
+        except IndexError:
+            bot.message(channel, "Oops! Looks like you didn't give enough arguments!")
```

Comment:
Maybe echo the usage here to prompt the user as to how it should look?

Reply:
will do